src/main/java/lu/oop/server/app/models/messages/MessageModel.java

```diff
@@ -0,0 +1,70 @@
+package lu.oop.server.app.models.messages;
+
+import java.sql.Timestamp;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.*;
+import lu.oop.server.app.models.users.*;
+
+@Entity
+@Table(name = "message")
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
```

Comment:
Doesn't do anything currently. In the future we need to decide the inheritance type. https://www.baeldung.com/hibernate-inheritance#joined-table joined table might be good choice, since messages - file, normal and so on are quite different. Having a 1 table with blob NULL and text NULL might be bad for performance.